tests/CommandLineTestRunnerTest.cpp

```diff
@@ -94,3 +95,25 @@ TEST(CommandLineTestRunner, NoPluginsAreInstalledAtTheEndOfARunWhenTheArgumentsA
 
     LONGS_EQUAL(0, registry.countPlugins());
 }
+
+struct TestOutputCheckingCommandLineTestRunner : public CommandLineTestRunner
+{
+    TestOutputCheckingCommandLineTestRunner(int ac, const char** av, TestOutput* output, TestRegistry* registry) :
+        CommandLineTestRunner::CommandLineTestRunner(ac, av, output, registry)
+    {
+    }
+
+    bool hasJUnitTestOutput(void)
+    {
+        return NULL != dynamic_cast<JUnitTestOutput*>(output_);
```

Comment:
Okay. I'll think about alternatives. The problem is, that CommandlineArguments would happily assert it's junit (because the commandline param was such) but I could remove the code I'm testing here (as I mistakenly did before) and the test would then happily pass...

One approach would be asserting that print() doesn't print anything.

It would be nice though if output_ itself could be queried for it's junit-ishness...